.github/workflows/integrate-and-deploy.yml

```diff
@@ -62,13 +62,9 @@ jobs:
           cache: 'pnpm'
       -
         run: pnpm install
-      -
-        name: Prep for production build of legacy app
-        # docker compose build doesn't allow passing a `--target` parameter, so we have to replace the target in docker-compose.yml
-        run: "sed -i 's/target: development/target: production/g' docker-compose.yml"
       -
         name: Build legacy app
-        run: docker compose build --build-arg ENVIRONMENT=production --build-arg BUILD_VERSION=${{ steps.image.outputs.TAG_APP }} app
+        run: docker compose -f docker-compose.yml -f docker-compose.production.yml build --build-arg ENVIRONMENT=production --build-arg BUILD_VERSION=${{ steps.image.outputs.TAG_APP }} app
```

Comment:
nice job! I think you can remove `--build-arg ENVIRONMENT=production` since it's defined already in the prod compose file, the build version is obviously still required

Reply:
Done in commit 02a1e51b3.